### PR TITLE
Simplify log_versions

### PR DESCRIPTION
**Pull request by cevich**
Previously versions of the various docker RPMS was collected in two
places. This is redundant possibly ambiguous, and unnecessary.  The
"active" version of docker is also meaningless since subtests are free
to restart at will.  It would only represent the active version when the
pretest runs.  Better to leave making any declaration of "active"
version up to any subtests where it matters (none currently).

Signed-off-by: Chris Evich <cevich@redhat.com>

### CODE REVIEW

**Comment by edsantiago** [comment]
> The "active" version of docker is also meaningless since subtests are free to restart at will

This isn't quite true. Subtests can restart, possibly with different docker options, but there is currently no mechanism for a test to switch between docker and docker-latest. This would be such an incredibly bad idea that I don't foresee us ever adding such functionality.

**Comment by cevich** [comment]
Which docker is currently running is the one it says when it prints out the version.

**Comment by edsantiago** [comment]
Can you rephrase that please? I don't understand it.

I will submit a PR to reinstate docker_rpm_active because my jenkins-analyze script relies on it.

**Comment by cevich** [comment]
[(2) [CI] Collect set of critical rpm NVRA's per run.](https://trello.com/c/JS4KG9H5/499-2-ci-collect-set-of-critical-rpm-nvras-per-run)

**Comment by edsantiago** [comment]
For benefit of anyone finding this thread in the future: what @cevich refers to in his above comment is that one could read `sysinfo/docker_version` and then match that to either `docker` or `docker-latest` in the file `sysinfo/key_rpms`. This doesn't actually work, because docker and docker-latest may be the same major version (e.g. different 1.12.x builds).

In the absence of `sysinfo/docker_rpm_active`, one way for a reader to determine the active docker is to fetch `sysinfo/sysconfig.tar.gz`, extract it into a tempdir, read the extracted `sysconfig/docker` file, and check the `DOCKERBINARY=` line near the bottom.
